handlers/email/email_test.go

```diff
@@ -104,6 +104,7 @@ func TestEmailHandler(t *testing.T) {
 	email.SetTimestampFormat("MST")
 	email.SetTemplate(defaultTemplate)
 	email.SetEmailConfig("localhost", 3041, "", "", "from@email.com", []string{"to@email.com"})
+	email.SetSend(true)
```

Comment:
It would be great if you could add an email test where it is set to false to ensure that when disabled it works appropriately.